sensor_msgs/PointCloud2 global_map
---
bool success